src/Microsoft.ML.Data/Transforms/SkipTakeFilter.cs

```diff
@@ -26,8 +26,23 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// Allows limiting input to a subset of row at an optional offset.  Can be used to implement data paging.
+    /// Allows limiting input to a subset of row at an optional offset. It can either take first <see cref="_take"/> rows and ignore all others.
+    /// Or it can skip first <see cref="_skip"/> rows and returns the rest.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Any |
+    /// | Output column data type | The same as the data type in the input column |
+    ///
+    /// ]]>
+    /// </format>
+    /// </remarks>
+    /// <seealso cref="Microsoft.ML.DataOperationsCatalog.SkipRows(Microsoft.ML.IDataView,System.Int64)" />
```

Comment:
>SkipRows [](start = 58, length = 8)

What's the point of all this if class for this summary is internal?

Reply:
also, I'd leave the filters out, since they don't change the schema. 

---
In reply to: [276867180](https://github.com/dotnet/machinelearning/pull/3415#discussion_r276867180) [](ancestors = 276867180)

Reply:
We don't need this 'internal' change. Also filters are different from transforms and don't return an estimator. they return an IDV.

The actual public API is here:
[https://github.com/dotnet/machinelearning/blob/909721e0cab3eb85b3e1cb4f010f6b086620517e/src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs#L352](https://github.com/dotnet/machinelearning/blob/909721e0cab3eb85b3e1cb4f010f6b086620517e/src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs#L352)

I see that Rogan has already polished them as part of the samples. I think they're in good shape. I'll close this PR. If you think otherwise, please reopen. 

---
In reply to: [276907066](https://github.com/dotnet/machinelearning/pull/3415#discussion_r276907066) [](ancestors = 276907066,276867180)